packages/cli/commands/project/dev.js

```diff
@@ -215,6 +232,62 @@ exports.handler = async options => {
     }
   }
 
+  if (createNewDeveloperTestAccount) {
+    let currentPortalCount = 0;
+    let maxTestPortals = 10;
```

Comment:
Default here should always be 10, but this will get overridden with whatever gets returned from BE which gets pulled from a config value